src/components/builder/loader.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import {Meta, StoryObj} from '@storybook/react';
+
+import Loader from '@/components/builder/loader';
+
+export default {
+  title: 'Formio/Builder/Loader',
+  component: Loader,
+  parameters: {
+    controls: {hideNoControlsWarning: true},
+    docs: {
+      source: {
+        type: 'dynamic',
+        excludeDecorators: true,
+      },
+    },
+    modal: {noModal: true},
+  },
+} as Meta<typeof Loader>;
+
+export const Default: StoryObj<typeof Loader> = {
+  render: () => <Loader />,
+  args: {},
+};
```

Comment:
```suggestion
type Story = StoryObj<typeof Loader>;

export const Default: Story = {};
```
defining the type alias outside makes it a bit more re-usable/DRY and the `render` override isn't needed since that's what SB does by default.

Reply:
Check, I've updated it